api/src/certification/configuration/infrastructure/repositories/active-calibrated-challenge-repository.js

```diff
@@ -7,18 +7,18 @@ import { ActiveCalibratedChallenge } from '../../domain/read-models/ActiveCalibr
 
 /**
  * @param {Object} params
- * @param {ComplementaryCertificationKeys} params.complementaryCertificationKey
+ * @param {ComplementaryCertificationKeys} params.scope
  * @param {number} params.calibrationId
  * @returns {Promise<Array<ActiveCalibratedChallenge>>}
  * @throws {NotFoundError}
  */
-export async function getByComplementaryKeyAndCalibrationId({ complementaryCertificationKey, calibrationId }) {
+export async function getByComplementaryKeyAndCalibrationId({ scope, calibrationId }) {
```

Comment:
```suggestion
export async function getByScopeAndCalibrationId({ scope, calibrationId }) {
```

Peut être fait dans une autre PR

Reply:
Done ✅